content/akeneo-web-api.yaml

```diff
@@ -2026,6 +2026,142 @@ paths:
+            "current_page": 1,
+            "items_count": 18,
+            _embedded: {
+              "items": [
+                {
+                  "_links": {
+                      "self": {
+                        "href": "https://demo.akeneo.com/api/rest/v1/measure-family/Area"
+                      }
+                    },
+                    "code": "Area",
+                    "standard": "SQUARE_METER",
+                    "units": [
+                      {
+                        "code": "SQUARE_MILLIMETER",
+                        "convert": {
+                          "mul": "0.000001"
+                        },
+                        "symbol": "mm\u00b2"
+                      },
+                      {
+                        "code": "SQUARE_CENTIMETER",
+                        "convert": {
+                          "mul": "0.0001"
+                        },
+                        "symbol": "cm\u00b2"
+                      },
+                      {
+                        "code": "SQUARE_DECIMETER",
+                        "convert": {
+                          "mul": "0.01"
+                        },
+                        "symbol": "dm\u00b2"
+                      },
+                      {
+                        "code": "SQUARE_METER",
+                        "convert": {
+                          "mul": "1"
+                        },
+                        "symbol": "m\u00b2"
+                      },
+                      {
+                        "code": "CENTIARE",
+                        "convert": {
+                          "mul": "1"
+                        },
+                        "symbol": "ca"
+                      },
+                      {
+                        "code": "SQUARE_DEKAMETER",
+                        "convert": {
+                          "mul": "100"
+                        },
+                        "symbol": "dam\u00b2"
+                      },
+                      {
+                        "code": "ARE",
+                        "convert": {
+                          "mul": "100"
+                        },
+                        "symbol": "a"
+                      },
+                      {
+                        "code": "SQUARE_HECTOMETER",
+                        "convert": {
+                          "mul": "10000"
+                        },
+                        "symbol": "hm\u00b2"
+                      },
+                  ]
+                }
+              ]
+            }
+          }
+        401:
+          $ref: "#/responses/401Error"
+        403:
+          $ref: "#/responses/403Error"
+        406:
+          $ref: "#/responses/406Error"
+  /api/rest/v1/measure-family/{code}:
+    get:
+      summary: Get a measure family
+      operationId: "measure_family_get"
+      description: This endpoint allows you to get the information about a given measure family.
+      tags:
+        - Measure Families
```

Comment:
Measure families (I don't know if uppercase is important or not, I don't know the usefulness of this tag actually)